resources/multisite_drupal_standard.make

```diff
@@ -1042,12 +1042,12 @@ libraries[respond][download][url] = https://raw.githubusercontent.com/scottjehl/
 projects[ec_resp][type] = theme
 projects[ec_resp][download][type] = git
 projects[ec_resp][download][url] = https://github.com/ec-europa/ec_resp.git
-projects[ec_resp][download][tag] = 2.3.8
+projects[ec_resp][download][tag] = 2.3.9
```

Comment:
The version 2.4 is blocked and only for bug-fixing. It's not necessary to upgrade ec_resp theme.

Reply:
Ha ????